app/react/portainer/environments/wizard/EnvironmentsCreationView/EnvironmentsCreationView.tsx

```diff
@@ -49,6 +52,11 @@ export function EnvironmentCreationView() {
   const envTypes = useParamEnvironmentTypes();
   const { trackEvent } = useAnalytics();
   const router = useRouter();
+  const isPodmanEnabledQuery = useFeatureFlag('podman');
+  // for the stepper, assume isPodmanEnabled is enabled, until shown otherwise
+  const isPodmanEnabled =
+    isPodmanEnabledQuery.data === undefined ? true : isPodmanEnabledQuery.data;
+  const environmentTypes = getEnvironmentTypes(isPodmanEnabled);
```

Comment:
In this case we should probably assume that the flag is disabled and not enabled by default. The purpose of a feature flag is to hide feature by default and only enable them on demand.

Also, use `FeatureId.PODMAN` or a custom hook (see above)

```ts
  const { data: isPodmanEnabled } = useFeatureFlag(FeatureId.PODMAN);
  const environmentTypes = getEnvironmentTypes(!!isPodmanEnabled);
```
OR
```ts
  const isPodmanEnabled = useIsPodmanEnabled();
  const environmentTypes = getEnvironmentTypes(isPodmanEnabled);
```

Reply:
There's a weird exception where isPodmanEnabledQuery is still loading. I updated the comment and code to hopefully reflect this case better